### ISSUE TITLE
not formatting on compile automatically

### ISSUE THREAD

**Issue by MarkRBM** [lucidsoftware/neo-sbt-scalafmt#2]
having just installed neo-sbt-scalafmt following the readme instructions and setting scalafmt version to 0.6.6 formating automatically before compile does not appear to be working, I tried setting scalafmtOnCompile := true explicitly and it still dosnt want to work.  sbt version is 0.13.12

**Comment by olafurpg** [comment]
Is `scalafmtOnCompile` enabled in all projects? You can try `scalafmtOnCompile in ThisBuild := true` and check the defined values with `show scalafmtOnCompile` inside sbt shell.

**Comment by diego-alvarez-hs** [comment]
It doesn't work for me either.
```
> show scalafmtOnCompile
[info] models/compile:scalafmtOnCompile
[info] 	true
[info] service/*:scalafmtOnCompile
[info] 	true
[info] sdk/compile:scalafmtOnCompile
[info] 	true
[info] integration-tests/compile:scalafmtOnCompile
[info] 	true
[info] root/*:scalafmtOnCompile
[info] 	true
```

**Comment by diego-alvarez-hs** [comment]
```
sbt.version=0.13.15
...
addSbtPlugin("com.lucidchart"    % "sbt-scalafmt"          % "0.3")
```

**Comment by MarkRBM** [comment]
show scalafmtOnCompile returns true

**Comment by pettyjamesm** [comment]
I'm seeing the same thing scalafmtOnCompile is set. Calling scalafmt explicitly in the root project aggregates to the sub-projects successfully, as does calling `subproject/scalafmt` and `subproject/test:scalafmt`, but calling compile and test:compile on the root project only compiles and does not trigger formatting.

**Comment by 2m** [comment]
Also noticed this. Even after `clean` compiling source code does not trigger reformat.

**Comment by jeantil** [comment]
same problem here 
```
sbt.version=0.13.15
```
```
addSbtPlugin("com.lucidchart" % "sbt-scalafmt" % "0.3")
```
```
enablePlugins(ScalafmtPlugin)
scalafmtVersion := "1.0.0-RC1"
```

**Comment by pauldraper** [comment]
Fixed in 1.0. Scopes needed adjusting. https://github.com/lucidsoftware/neo-sbt-scalafmt/commit/20890d7b39ed708dc30e763e2a0ed6eb85844e73
